src/main/java/edu/uw/cse403/Account.java

```diff
@@ -0,0 +1,22 @@
+package edu.uw.cse403;
+
+public class Account {
+    double principal,rate; int daysActive,accountType;
+    public static final int STANDARD = 0, BUDGET = 1, PREMIUM = 2, PREMIUM_PLUS = 3;
+
+    public static double calculateFee(Account[] accounts)
+    {
+        double totalFee = 0.0;
+        Account account;
+        for (int i=0;i<accounts.length;i++) {
+            account=accounts[i];
+            if(account.accountType == Account.PREMIUM ||
+               account.accountType == Account.PREMIUM_PLUS)
+                totalFee += .0125 * (   // 1.25% broker's fee
+                        account.principal * Math.pow(account.rate,
+                        (account.daysActive / 365.25))
+                        - account.principal);   // interest-principal
```

Comment:
This is line is very long, let's break this apart into smaller, easier to understand portions.